src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -145,9 +144,8 @@ class Decimal extends React.Component {
   };
 
   removeDelimiters = (value) => {
-    const format = I18nHelper.format();
-    const delimiter = `\\${format.delimiter}`;
-    const delimiterMatcher = new RegExp(`[${delimiter}]*`, "g");
+    const delimiter = `\\${this.getSeparator("group")}`;
+    const delimiterMatcher = new RegExp(`[${delimiter} ]*`, "g");
```

Comment:
```suggestion
    const delimiterMatcher = new RegExp(`[\\${this.getSeparator("group")} ]*`, "g");
```

lets combine these lines :)